docs/pydbus.rst

```diff
@@ -0,0 +1,91 @@
+
+* Lazy DBus connections: DBus connections are established on demand.
+
+* Lazy DBus proxies: Attributes of DBus proxies are created on demand.
+
+
+What is different
+-----------------
+
+* No context managers: There are no context managers in dasbus. Context managers and event
+  loops don't work very well together.
+
+* No auto-completion: There is no support for automatic completion of DBus names and paths.
+  We recommend to work with constants defined by classes from :mod:`dasbus.identifier`
+  instead of strings.
+
+* No unpacking of variants: The dasbus library doesn't unpack variants by default. It means
+  that values received from DBus match the types declared in the XML specification. Use the
+  :func:`get_native <dasbus.typing.get_native>` function to unpack the values.
+
+* Obtaining proxy objects: Call the :meth:`get_proxy <dasbus.connection.MessageBus.get_proxy>`
+  method to get a proxy of the specified DBus object.
+
+* No single-interface view: DBus proxies don't support single-interface views. Use the
+  :class:`InterfaceProxy <dasbus.client.proxy.InterfaceProxy>` class to access a specific
+  interface of a DBus object.
+
+* Higher priority of standard interfaces: If there is a DBus interface in the XML specification
+  that redefines a member of a standard interface, the DBus proxy will choose a member of the
+  standard interface. Use the :class:`InterfaceProxy <dasbus.client.proxy.InterfaceProxy>` class
+  to access a specific interface of a DBus object.
+
+* No support for help: Members of DBus proxies are created lazily, so the build-in ``help``
+  function doesn't return useful information about the DBus interfaces.
+
+* Watching DBus names: Use :class:`a service observer <dasbus.client.observer.DBusObserver>`
+  to watch a DBus name.
+
+* Acquiring DBus names: Call the :meth:`register_service <dasbus.connection.MessageBus.register_service>`
+  method to acquire a DBus name.
+
+* Providing XML specifications: Use the ``__dbus_xml__`` attribute to provide the XML
+  specification of a DBus object. Or you can generate it from the code using the
+  :func:`dbus_interface <dasbus.server.interface.dbus_interface>` decorator.
+
+* No support for polkit
+
+What is the same (for now)
+--------------------------
+
+* No support for other event loops: Dasbus uses GLib as its backend, so it requires to use
+  the GLib event loop. However, the GLib part of dasbus is separated from the rest of the code,
+  so it shouldn't be too difficult to add support for a different backend. It would be necessary
+  to replace :class:`dasbus.typing.Variant` and :class:`dasbus.typing.VariantType` with their
+  abstractions and reorganize the code.
+
+* No support for org.freedesktop.DBus.ObjectManager: There is no support for object managers,
+  however the :class:`DBus containers <dasbus.server.container.DBusContainer>` could be a good
+  starting point.
+
+* No support for unix file descriptors
```

Comment:
```suggestion
* No support for unix file descriptors.
```

Reply:
Similar question here? Do you mean sending FDs by DBus and understanding the type? If yes, then it should be easy to use just `int` to send it.

Reply:
DBus has a special data type for file descriptors and it is necessary to call some special GLib functions. There is an open issue for pydbus, so it is fair to mention it here.

Reply:
Same here.

Reply:
This item has changed as well.